pkg/porter/testdata/runs/expected-output.txt

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
Something about how the test is being set up isn't quite right. This file should be the output that someone would see if they ran the command and didn't set the format (i.e. what you see in your screenshot). It shouldn't be json. 

Same for the yaml output test case.